Resources/templates/responsive/channel/call/layout.php

```diff
@@ -14,13 +14,14 @@
 <link rel="stylesheet" type="text/css" href="<?= SRC_URL ?>/assets/vendor/slick-carousel/slick/slick-theme.css"/>
 
 <?php
-
 $this->append();
 
 $this->section('header');
+?>
 
-//Header empty
+<?php $this->insert('channel/call/partials/banner_header'); ?>
```

Comment:
Why is the php closing tag in line 20 if we reopen php in line 22 with no non-php code in between? Same for the php closing tag in this line and the next.